frontend/apps/marketing/src/components/section/index.tsx

```diff
@@ -0,0 +1,28 @@
+import '@code-dot-org/component-library/cms/section/index.css';
+
+// Import the background image used in the pattern backgrounds on Contentful
+import bgPatternImage from '../../../public/images/bg-pattern-lines.png';
```

Comment:
We should be able to improve this pathing without using relative paths. 

```suggestion
import bgPatternImage from '@public/images/bg-pattern-lines.png';
```

Could you also add this to `tsconfig.json`?

```
     "paths": {
+      "@public/*": ["./public/*"],
       "@/*": ["./src/*"]
     },
```

Reply:
Yay, updated!